sky/utils/cli_utils/status_utils.py

```diff
@@ -325,6 +327,14 @@ def _get_resources(cluster_record: _ClusterRecord) -> str:
         if (handle.launched_nodes is not None and
                 handle.launched_resources is not None):
             launched_resource_str = str(handle.launched_resources)
+            # accelerator_args is way too long.
+            # Convert from:
+            #  GCP(n1-highmem-8, {'tpu-v2-8': 1}, accelerator_args={'runtime_version': '2.5.0'}  # pylint: disable=line-too-long
+            # to:
+            #  GCP(n1-highmem-8, {'tpu-v2-8': 1}...)
+            pattern = ', accelerator_args={.*}'
+            launched_resource_str = re.sub(pattern, '...',
+                                           launched_resource_str)
```

Comment:
Seems if we specify the `disk_type`, `disk_tier`, etc, they will be after `...`. Is that intended?

Reply:
Yes, it'd show `GCP(n1-highmem-8, {'tpu-v2-8': 1}..., cpus=2+, dist_tier=high)`, etc. Wdyt?